packages/ripple-ui-components/components/Organisms/ImageGallery/FullscreenImage.vue

```diff
@@ -40,6 +41,11 @@ export default {
     return {
       toggleCaption: false
     }
+  },
+  computed: {
+    isCaptionShowing () {
+      return (this.caption && this.toggleCaption) || !this.caption
```

Comment:
I feel this logic is a little confusing... The second part, if there is no caption, then we show it?

Reply:
In the caption panel exists the title and the caption.

If we have a caption prop
  then we want a button to show / hide the caption panel depending on the toggle state.
If no caption prop was given
  then we want want that caption panel to always display (so we can see the title)